### PR TITLE
Add migration to remove invalid tximport associations

### LINKED ISSUE #2054: Processing info: Sample is not available but only tximport is run on it
Ex:
https://www.refine.bio/experiments/SRP074593/gene-expression-profiling-of-drosophila-s2r-cells-following-rnai-mediated-knockdown-of-transcription-factors

Screen Shot 2020-01-03 at 2 58 39 PM

We should just show the submitter supplied info, since the samples aren't really processed through our pipeline

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

close #2054 

## Purpose/Implementation Notes

Removes associations between tximport computational results and samples that are not processed.

## Types of changes

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests

Ran migrations locally

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: common/data_refinery_common/migrations/0052_remove_invalid_tximport_associations.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from django.db import migrations, models
```
Comment (davidsmejia) [inline common/data_refinery_common/migrations/0052_remove_invalid_tximport_associations.py]:
```suggestion
from django.db import migrations
```

File: common/data_refinery_common/migrations/0052_remove_invalid_tximport_associations.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from django.db.models import Count, F
```
Comment (davidsmejia) [inline common/data_refinery_common/migrations/0052_remove_invalid_tximport_associations.py]:
```suggestion

```

Comment (arielsvn) [reply common/data_refinery_common/migrations/0052_remove_invalid_tximport_associations.py]: removed, thanks!

**Comment by arielsvn** [comment]
This will remove `11671` associations in total:

```
>>> SampleResultAssociation.objects.filter(sample__is_processed=False, result__processor__name__iexact="tximport").count()
11671
```

**Pull request merged**